src/OSL-2.1.xml

```diff
@@ -0,0 +1,199 @@
+<spdx name="Open Software License 2.1" identifier="OSL-2.1" osi-approved="true">
+  <urls>
+    <url>http://opensource.org/licenses/OSL-2.1</url>
+    <url>http://web.archive.org/web/20050212003940/http://www.rosenlaw.com/osl21.htm</url>
+  </urls>
+  <notes>Same as version 2.0 of this license except with changes to section 10</notes>
+  <license>
+    <title>
+      <p>The Open Software Licensev. 2.1</p>
+    </title>
+    <optional>
```

Comment:
not 100% sure about the substantive question on this one, but I agree we shouldn't make it optional right now. let's leave it required for now.